drinfomon.lic

```diff
@@ -852,6 +852,7 @@ while line = script.gets
         end
       end
     end
+    fput('info') if DRStats.guild.nil? && !dead?
```

Comment:
Just realized the command above is using 'put' not 'fput'. Should I change this around? Not familiar with `put`

Reply:
`fput` will wait for a response from the game, `put` is more appropriate here

Reply:
Got it & fixed